.gitmodules

```diff
@@ -1,3 +1,9 @@
 [submodule "libafl_concolic/symcc_runtime/symcc"]
 	path = libafl_concolic/symcc_runtime/symcc
 	url = https://github.com/AFLplusplus/symcc.git
+[submodule "fuzzers/verilator_cva6/cva6"]
```

Comment:
IIRC we preferred to `git clone` in the makefile, since, otherwise, referencing to the libafl repo from Cargo.toml downloads all subrepos, making it rather slow

Reply:
yes I was about to say the same thing

Reply:
I believe the fuzzers are marked as excluded, so in this case it should not cause this issue.